content/manager/openstack-reference.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: bt_wiki
+title: Openstack Bootstrap
```

Comment:
Probably needs a different name. 'Openstack inputs and resources'?

What about the file name?